test/e2e-tests.sh

```diff
@@ -385,6 +395,20 @@ function test_consolidated_channel_tls() {
   uninstall_channel_crds || return 1
 }
 
+function test_consolidated_channel_sasl() {
+  # Test the consolidated channel with SASL
+  echo "Testing the consolidated channel with SASL"
+  # Set the URL to the SASL listeners config
+  cp ${KAFKA_SASL_CONFIG} "${CONSOLIDATED_TEMPLATE_DIR}/configmaps/kafka-config.yaml"
+  KAFKA_CLUSTER_URL=${KAFKA_SASL_CLUSTER_URL}
+
+  install_consolidated_channel_crds || return 1
+
+  go_test_e2e -tags=e2e -timeout=40m -test.parallel=${TEST_PARALLEL} ./test/e2e -channels=messaging.knative.dev/v1beta1:KafkaChannel  || fail_test
+
+  uninstall_channel_crds || return 1
+}
+
```

Comment:
Do we really need to run the whole suite for testing tls and sasl? Could we just run a single test case?

Reply:
I'd really like to make sure the entire tests run, to have some load, and robustness 

Instead of "chaining" them, we can move them to three different jobs, with a different PR? (requires work on test-infra)

Reply:
yes this sounds like a good idea! This way, we also parallelize and make the build faster overrall

Reply:
> yes this sounds like a good idea! This way, we also parallelize and make the build faster overrall

yes! Just wanted it to work first, before getting fancy